client/components/__tests__/shortener_test.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { StoreProvider, createStore, Thunk, thunk } from "easy-peasy";
+import userEvent from "@testing-library/user-event"
+import { store } from "../../store";
+import { links, Link, Links, NewLink } from "../../store/links";
```

Comment:
These are unused and can be removed. There are some other unused variables and arguments too can you check them as well.